jenkins/ci.suse.de/pipelines/cloud-gating.Jenkinsfile

```diff
@@ -35,7 +36,12 @@ pipeline {
 
           env.starttime = sh (
             returnStdout: true,
-            script: 'echo $(date +%s)'
+            script: '''
+              rfcdate="$( curl -sI $staging_url | grep 'Last-Modified: ' | head -n1 | cut -d' ' -f2- )"
+              epoch=$(date +%s -d "$rfcdate")
+              test "$epoch" -gt "1400000000"
```

Comment:
not sure why you are doing this test. Should we throw an exception if Last-Modified's unix timestamp is not what we expect instead? Could you clarify?

Reply:
Yes, that is a check to see if the date is sane (later than somewhen in 2014). If it is not sane the whole job should abort and fail.

Reply:
maybe add a comment to make it clear to future us